### DISCUSSION TITLE
Visibility of Github Pages

### DISCUSSION THREAD

**Discussion by kkhandelwal** [community/community#33274] [Other Feature Feedback, Questions, & Ideas] [answered]
How to control Visibility of Github Pages? if its paid - price to get this functionality ?

**Comment by melvincwng** [comment, marked as answer]
Hey @kkhandelwal, with regards to your question, I suggest you refer to this official GitHub Documentation link on [how to control visibility of GitHub pages](https://docs.github.com/en/enterprise-cloud@latest/pages/getting-started-with-github-pages/changing-the-visibility-of-your-github-pages-site).

Scroll down to the section titled 'Changing the visibility of your GitHub Pages site', and refer to the 5 steps as stated in the documentation website, to change your GitHub pages visibility.

Do take note of the following, as quoted from the documentation website:
> **Note**
Note: To publish a GitHub Pages site privately, your organization must use GitHub Enterprise Cloud. For more information about how you can try GitHub Enterprise Cloud for free, see "[Setting up a trial of GitHub Enterprise Cloud](https://docs.github.com/en/enterprise-cloud@latest/get-started/signing-up-for-github/setting-up-a-trial-of-github-enterprise-cloud)."

Hope this answers your question :)

**Comment by douglaslassance** [comment]
Hard to justify paying that much more per user for that feature. I wish it was included with the "Team" pricing.

**Reply by mschoettle** [reply]
I agree! I created a product feedback discussion with the request here: https://github.com/orgs/community/discussions/202856 Would be great to get it upvoted.
